.gitignore

```diff
@@ -0,0 +1,43 @@
+# Whisk
```

Comment:
All right. Currently I think we only need to keep 
# Gradle
build/
We should remove the rest.